sass/components/html/listing__wrapper--row-three.html

```diff
@@ -1,7 +1,7 @@
 <div class="listing__wrapper listing__wrapper--row-three">
   <ul class="listing listing--no-border listing--teaser listing--cta">
     <li class="listing__item">
-      <div class="listing__item__wrapper">
```

Comment:
To explain you, the real thing here is the fact that the component is not "fully" implemented in the info website. In particular you don't have the listing__item__wrapper element there. Therefore it was breaking now that i removed the .node class from the sass file. 
We decided to hotfix like this, without making changes in the theming of the component which is still what we really need to do.
In this example, basically, we alternate cases where you have that element and cases where you don't, just to prove that it works in both the situation.
If you feel this is ambiguous, i can understand and i wouldn't have problems removing the two examples here, i just wanted to explain you the reason for this.